pulsar-proxy/pom.xml

```diff
@@ -110,6 +110,24 @@
       <artifactId>simpleclient_servlet</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+         </exclusion>
+      </exclusions>
+    </dependency>
+    
+    <!-- httpclient uses it for logging --> 
+    <dependency>
+      <groupId>commons-logging</groupId>
+      <artifactId>commons-logging</artifactId>
+      <version>1.1.1</version>
```

Comment:
We shouldn't need version here.

Reply:
that's true, it's already part of `pulsar-client-original` so, need to include here.